src/dto/sheet_for_list.rs

```diff
@@ -0,0 +1,17 @@
+use SheetId;
+
+#[derive(Debug, Deserialize)]
+pub struct SheetForList {
+    id: SheetId,
+    name: String
+}
+
+impl SheetForList {
+    pub fn get_name(&self) -> &str {
```

Comment:
java style accessors? what's the reason to do accessors instead of direct field access?

Reply:
Only encapsulation, that's the only reason. And it costs 4 lines of trivial code.